packages/handlebars/src/helpers/path.js

```diff
@@ -29,10 +29,9 @@ module.exports = function (fractal) {
             return path;
         }
 
-        return utils.relUrlPath(
-            path,
-            _.get(root._env.request || root._request, 'path', '/'),
-            fractal.web.get('builder.urls')
-        );
+        return utils.relUrlPath(path, _.get(root._env.request || root._request, 'path', '/'), {
+            ...fractal.web.get('builder.urls'),
+            relativeToCurrentFolder: true,
```

Comment:
Regarding this - maybe it would be better to actually define this in the default config as such? Then it can be turned on/off in the project config, if need be (in case this actually breaks something for someone).

This would be the correct place: https://github.com/frctl/fractal/blob/main/packages/fractal/config.js#L117

Reply:
Yes - much better - thanks

Updated in [1d4d1af](https://github.com/frctl/fractal/pull/1062/commits/1d4d1afffbae6fad5bfb91fd66f7371fb1cf9ad1)